### PR TITLE
Update `test_positive_all_packages_update`

### PR DESCRIPTION
**Pull request by mstrlc**
Automate [BZ 2073535](https://bugzilla.redhat.com/show_bug.cgi?id=2073535) (`satellite-maintain packages check-update fails when there are no packages to be updated`).

### CODE REVIEW

**Comment by mstrlc** [comment]
```
trigger: test-robottelo
pytest: tests/foreman/maintain/test_packages.py::test_positive_fm_packages_check_update
```

**Comment by mstrlc** [comment]
```
trigger: test-robottelo
pytest: tests/foreman/maintain/test_packages.py::test_positive_fm_packages_check_update
```

File: tests/foreman/maintain/test_packages.py [hunk 1]
```diff
@@ -289,3 +289,65 @@ def test_positive_fm_packages_sat_installer(sat_maintain):
...
+
+    # Run satellite-maintain packages update
+    result = sat_maintain.cli.Packages.update(packages='walrus', options={'assumeyes': True})
+    assert 'OK' in result.stdout
+    assert result.status == 0
+
+    # Verify that there are no packages available for update and return code is 0
+    result = sat_maintain.cli.Packages.check_update()
+    assert 'FAIL' not in result.stdout
+    assert result.status == 0
```
Comment (Gauravtalreja1) [inline tests/foreman/maintain/test_packages.py:353]: This test scenario looks very identical to `test_positive_fm_packages_update`, can't we add this additional check of check-update when no update is available, into above test after package is updated?

**Review by lhellebr — COMMENTED** [review]
I agree with both Gaurav and Griffin and I'm in favor of closing this. Unless there's some additional value to this test that I'm not aware of.

But we may want to add `@pytest.mark.include_capsule` to Gaurav's test.

**Comment by Griffin-Sullivan** [comment]
I think a good resolution for this PR would be to add the capsule marker like @lhellebr said, and finish the work I started in the other PR that got merged. The test still doesn't completely verify that there are no updates available at the end. See https://github.com/SatelliteQE/robottelo/pull/12009#discussion_r1276693357. @mstrlc if you don't want to add those updates yourself I'm willing to do it and push them here. Up to you.

**Comment by mstrlc** [comment]
Sure, I will look into it & update this test. Sorry, kinda forgot that I had it open.

**Comment by mstrlc** [comment]
Updated Griffin's test to add the capsule marker and a check if the output of the `Packages.check_update()` actually doesn't contain any packages, as the return code is same whether there are any to update or not.

**Comment by mstrlc** [comment]
```
trigger: test-robottelo
pytest: tests/foreman/destructive/test_packages.py::test_positive_all_packages_update
```

**Comment by mstrlc** [comment]
```
trigger: test-robottelo
pytest: tests/foreman/destructive/test_packages.py::test_positive_all_packages_update
```

**Review by Griffin-Sullivan — APPROVED** [review]
Works for me!

**Pull request merged**